src/privileged/multipreffer/api.js

```diff
@@ -29,21 +29,9 @@ this.multipreffer = class extends ExtensionAPI {
 };
 
 this.FirefoxHooks = {
-  init() {
-    AddonManager.addAddonListener(this);
-  },
-
-  get abortedPref() {
-    return `extensions.multipreffer.${gExtension.id}.aborted`;
-  },
-
+  _oldDefaultValues: {},
   async studyReady(studyInfo) {
```

Comment:
This function is run once per startup without any conditions that might prevent it, I take it?

Reply:
Yeah, we listen for the studyReady event from shield utils and that's when this gets called. We need to wait for this event in order to know which variation we're in.